js/kucoinfutures.js

```diff
@@ -1304,7 +1304,7 @@ module.exports = class kucoinfutures extends kucoin {
             'datetime': this.iso8601 (timestamp),
             'currency': code,
             'amount': amount,
-            'fromAccount': 'futures',
+            'fromAccount': 'future',
```

Comment:
i don't think we should change this instance, let's only change the `.market` attributes

Reply:
I think we should use one term throughout

Reply:
but I can add `fromAccount !== 'futures'` to the if

Reply:
> I think we should use one term throughout

I agree, but we can't change it all at once without introducing bugs, so first let's try to unify the markets and then we unify the defaultTypes and the accountTypes